### DISCUSSION TITLE
OOT Diffusion open source clothing replacer

### DISCUSSION THREAD

**Discussion by Maki9009** [light-and-ray/sd-webui-replacer#48] [Ideas]
https://huggingface.co/levihsu/OOTDiffusion

Came out recently with very good results id say. 
Would be nice to have

**Comment by light-and-ray** [comment]
It looks like face swapping but for clothes. I think you should share this idea here https://github.com/Gourieff/sd-webui-reactor. I don't see it fits for replacer, because it requires additional image
